sourcegraph/sourcegraph
# Adds domain column to github_apps migration

Adds new `domain` column to the github_apps table to close #52264
![image](https://github.com/sourcegraph/sourcegraph/assets/59381432/07ae36b9-acf7-4525-a460-20b7567dd4fc)

@kopancek - What default value do you want for this field for your use case? We will change the column to `batches` for Batch Changes GitHub Apps. We set the default value to `repos` for now, let us know what you think!  🙏🏻 

## Test plan
- Run `sg migration up` ensure new `domain` column to `github_apps` table
- Run `sg migration undo` new column is removed
